dataland-frontend/src/components/pages/RequestedDatasetsPage.vue

```diff
@@ -0,0 +1,452 @@
+            <span class="w-3 p-input-icon-left" style="margin: 15px">
+              <i class="pi pi-search pl-3 pr-3" aria-hidden="true" style="color: #958d7c" />
+              <InputText
+                data-test="requested-Datasets-searchbar"
+                v-model="searchBarInput"
+                placeholder="Search by company name"
+                class="w-12 pl-6 pr-6"
+              />
+              <i v-if="waitingForSearchResults" class="pi pi-search pl-3 pr-3" aria-hidden="true"></i>
+            </span>
+            <FrameworkDataSearchDropdownFilter
+              v-model="selectedFrameworks"
+              ref="frameworkFilter"
+              :available-items="availableFrameworks"
+              filter-name="Framework"
+              data-test="requested-Datasets-frameworks"
+              filter-id="framework-filter"
+              filter-placeholder="Search frameworks"
+              class="ml-3"
+              style="margin: 15px"
+            />
+            <div class="flex align-items-center">
+              <span
+                data-test="reset-filter"
+                style="margin: 15px"
+                class="ml-3 cursor-pointer text-primary font-semibold d-letters"
+                @click="resetFilters"
+                >RESET</span
+              >
+            </div>
+          </span>
+        </div>
+        <div class="col-12 text-left p-3">
+          <div class="card">
+            <DataTable
+              :value="displayedData"
+              class="table-cursor"
+              :rowHover="true"
+              data-test="requested-Datasets-table"
+              paginator
+              paginator-position="bottom"
+              :rows="datasetsPerPage"
+              lazy
+              :total-records="numberOfFilteredRequests"
+              @page="onPage($event)"
+              @sort="onSort($event)"
+            >
+              <Column header="COMPANY" class="d-bg-white w-2 qa-review-id" field="companyName" :sortable="true">
+                <template #body="slotProps">
+                  {{ slotProps.data.companyName }}
+                </template>
+              </Column>
+              <Column header="FRAMEWORK" class="d-bg-white w-2 qa-review-framework" :sortable="true" field="dataType">
+                <template #body="slotProps">
+                  <div>
+                    {{ getFrameworkTitle(slotProps.data.dataType) }}
+                  </div>
+                  <div
+                    data-test="framework-subtitle"
+                    v-if="frameworkHasSubTitle(slotProps.data.dataType)"
+                    style="color: gray; font-size: smaller; line-height: 0.5; white-space: nowrap"
+                  >
+                    <br />
+                    {{ getFrameworkSubtitle(slotProps.data.dataType) }}
+                  </div>
+                </template>
+              </Column>
+              <Column
+                header="YEAR"
```

Comment:
YEAR -> REPORTING PERIOD
In der Zukunft können das auch mal Quartale oder gar nur Monatsberichte sein